carapace-server/src/main/java/org/carapaceproxy/core/ProxyRequestsManager.java

```diff
@@ -428,6 +433,22 @@ public Publisher<Void> forward(ProxyRequest request, boolean cache) {
                     }
                     addCustomResponseHeaders(request, request.getAction().customHeaders);
 
+                    if (currentConfiguration.isHttp10BackwardCompatibilityEnabled() &&
```

Comment:
Maybe it's better to call parent.getCurrentConfiguration().isHttp10BackwardCompatibilityEnabled(), this way we don't need to call reloadConfiguration which destroys all connections pool

Reply:
@pv3nturi Right, make sense